assets/src/edit-story/app/media/useMediaReducer/reducer.js

```diff
@@ -83,6 +85,37 @@ function reducer(state, { type, payload }) {
       };
     }
 
+    case types.ADD_PROCESSING: {
+      const { videoId } = payload;
+      return {
+        processing: state.processing.push(videoId),
+        ...state,
+      };
+    }
```

Comment:
Although this code produces the desired effect (adds new id), but the way it does it is not desired, because:
1. `push` returns a new array length
2. it breaks immutable state principle (`state.processing === returnedState.processing`)

Reply:
Good catch, just being lazy. Should be fixed in https://github.com/google/web-stories-wp/pull/553/commits/f916af48b4b360a6956f9265eb1b7b6ff92ded31